### ISSUE TITLE
Label.Text is ruining my multi lines text

### ISSUE THREAD

**Issue by jrahma** [xamarin/Xamarin.Forms#8030]
Hi,

I am using below to get Multi-line Label which is working fine except it also consider the spaces before the text although it's just given for XAML align purpose and not for paragraph alignment just like all the other XAML indents

The result of the code is also attached as screenshot

```
<Grid HorizontalOptions="FillAndExpand" VerticalOptions="FillAndExpand" Padding="15">
    <Label>
        <Label.Text>
            Channel is your ads area where you publish all your ads.
            If you are an individual then channel name can be your name.
            If you are a business owner then you can use your business name.
            You can have more than one channel if you own more than one business and if you don;t want to list all ads under the same name.
        </Label.Text>
    </Label>
</Grid>

```

![Screenshot_20191016_015445_com softnames zeeraapp](https://user-images.githubusercontent.com/6236241/66873025-7d7ee580-efb8-11e9-9cb1-b1a76c597fba.jpg)

**Comment by StephaneDelcroix** [comment]
All white spaces, of any type, of any length, should be treated as a single white-space, unless wrapped in CDATA section. But you'll loose your CR in the process, too. Is that what you'd like ? how does it behave on UWP or WPF ?

**Comment by jrahma** [comment]
I don't have UWP nor WPF 

It should not add all this space when I just indent it for arranging purpose only

**Comment by StephaneDelcroix** [comment]
could you please rephrase this as an enhancement ? thanks

**Comment by StephaneDelcroix** [comment]
related to #6934

**Comment by kingces95** [comment]
I think that's just the way it goes when programming in XAML. Inside that tag the proceeding white space is not ignored. It's not unreasonable. For example, suppose that screenshot is actually what you _wanted_ to produce? If we ignored the whitespace how then would you reproduce that copy?

**Comment by jfversluis** [comment]
Thanks for this suggestion! As Xamarin.Forms is now in maintenance mode, this will not happen anymore for Xamarin.Forms. We're only adding bugfixes and stability fixes.

If this is still important to you, make sure to check the [.NET MAUI repo](https://github.com/dotnet/maui) and see if it's already on the roadmap. If not, feel free to open a discussion to discuss a change first or open an issue with a detailed feature request. Thanks!
